versions/1.21.5/src/main/java/at/hannibal2/skyhanni/features/foraging/ForagingTracker.kt

```diff
@@ -192,6 +195,26 @@ object ForagingTracker : SkyHanniBucketedItemTracker<ForagingTrackerLegacy.TreeT
         addItem(treeType, STRETCHING_STICKS, change, command = false)
     }
 
+    private var bonusDropsUncommonDropsList = listOf<NeuInternalName>()
+    private var bonusDropsEnchantedBooksList = listOf<NeuInternalName>()
+    private var bonusDropsBoostersList = listOf<NeuInternalName>()
+    private var bonusDropsShardsList = listOf<NeuInternalName>()
+    private var bonusDropsMobsList = listOf<String>()
+    private var bonusDropsRunesList = listOf<NeuInternalName>()
+    private var bonusDropsMiscDropsList = listOf<NeuInternalName>()
+
+
+    @HandleEvent
+    fun onRepoReload(event: RepositoryReloadEvent) {
+        val dropsJson = event.getConstant<TreeGiftBonusDropsJson>("foraging/TreeGiftBonusDrops")
+        bonusDropsUncommonDropsList = dropsJson.uncommonDrops
```

Comment:
Instead of creating the 7 lists in the file, you can define the `dropsJson` variable outside the repo reload function.
Then, you can access the data in the other functions with e.g. `dropsJson.uncommonDrops` instead of with `bonusDropsUncommonDropsList`.